doc/decisions/change_tracking.md

```diff
@@ -0,0 +1,147 @@
+# Change Tracking
+
+## Problem
+
+Currently, a range of Elektra plugins are implementing some sort of change-tracking for configuration data.
```

Comment:
change-tracking still has multiple spellings. If the word itself is written with `-`, then `-` always must be present (e.g. title above) and you can add it to scripts/sed.